src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/BlobStorageReadTest.java

```diff
@@ -43,7 +59,35 @@ public void connects_without_exceptions() throws URISyntaxException, StorageExce
         CloudBlobContainer abc = cloudBlobClient.getContainerReference("abc");
         abc.createIfNotExists();
 
+        Throwable throwable = catchThrowable(() -> {
+            BlobStorageRead blobStorageRead = new BlobStorageRead(cloudBlobClient, pdfsConsumer);
+            blobStorageRead.readBlobs();
+        });
+
+        assertThat(throwable).doesNotThrowAnyException();
```

Comment:
You have to put the lambda as the argument, not the result. And shouldn't it be `assertThatCode(...`?